tests/templates/icmp_responder.conf.j2

```diff
@@ -1,7 +1,6 @@
 [program:icmp_responder]
 command=/root/env-python3/bin/python /opt/icmp_responder.py {{ icmp_responder_args }}
 process_name=icmp_responder
-environment=PATH="/root/env-python3/bin"
```

Comment:
Why remove this line and keep `/root/env-python3/bin/python` in line 2? If the purpose is to use whatever default python binary, then line 2 should be updated as well. Eventually `/root/env-python3` may be removed in "py3only" ptf image.

Reply:
The `environment` is not required for the script to run. To keep backporting simpler I have done away from modifying the path to invoke Python 3. This PR enables that https://github.com/sonic-net/sonic-buildimage/pull/19813. Can you please review this too.